### ISSUE TITLE
TypeScript: Rename refactoring stops at destructured objects

### ISSUE THREAD

**Issue by jumpinjackie** [Microsoft/vscode#6227]
- VSCode Version: 1.1.0-insider
- OS Version: Windows 8.1 x64

Steps to Reproduce:
1. [Destructure](https://basarat.gitbooks.io/typescript/content/docs/destructuring.html) an object of some known TS class or interface
2. Rename a member in the class or interface that is referenced in the destructuring statement

Expect: The destructured member name to be updated to matched the renamed symbol
Get: The destructured member is not updated, causing a TS compilation error

See attached gif

![typescript_rename_destructuring](https://cloud.githubusercontent.com/assets/563860/15135901/015834a0-16bf-11e6-88dc-2dbc8ef523da.gif)

**Comment by mjbvz** [comment]
This seems to be fixed in VSCode 1.7.2:

![dec-01-2016 16-31-10](https://cloud.githubusercontent.com/assets/12821956/20818603/9bae50c8-b7e3-11e6-8d15-d279aa704c73.gif)


```tsx
import * as React from 'react';

interface IFooProps {
    name: string
    message: string
}

class Foo extends React.Component<IFooProps, any> {
    constructor(props) {
        super(props)
    }

    render(): JSX.Element {
        const {name, message} = this.props
        return <div>{message}</div>
    }
}  
```

Please give a more recent build a try and let me know if you still see this issue.

Thanks

**Comment by jumpinjackie** [comment]
Can confirm it is fixed on current VSCode (1.7.2). Thanks.
